access/resource_permissions.go

```diff
@@ -366,32 +365,30 @@ func ResourcePermissions() *schema.Resource {
 	}
 	return &schema.Resource{
 		Schema: s,
-		CustomizeDiff: customdiff.Sequence(
-			func(ctx context.Context, diff *schema.ResourceDiff, m interface{}) error {
-				me, err := identity.NewUsersAPI(ctx, m).Me()
-				if err != nil {
-					return err
+		CustomizeDiff: func(ctx context.Context, diff *schema.ResourceDiff, m interface{}) error {
```

Comment:
can you revert the formatting for this file not to interfere with diff?

Reply:
it's not a pure formatting, it's removal of `customdiff.Sequence`, just applying the earlier comment to multiple places.  But I can remove it